### PR TITLE
Add Gule Sider as impossible

### PR DESCRIPTION
**Pull request by Odyssey346**
This is a Norwegian "Yellow Pages" website. Very easy way to get yourself doxxed. Worst part is, it's hard to get your info off of there

### CODE REVIEW

**Comment by Odyssey346** [comment]
Don't merge yet, I just forgot to add something, and editing on mobile is pain

**Comment by Odyssey346** [comment]
All good.

**Review by tupaschoal — COMMENTED** [review]
Thanks for the contribution. One question though, it seems like though you can edit a lot of your stuff, you can't really delete the account or anonymise yourself. Is that so?

Because then I think it should be impossible.

**Comment by Odyssey346** [comment]
Yeah, you can't completely delete yourself from there

File: _data/sites.json [hunk 1]
```diff
@@ -5223,6 +5223,16 @@
         ]
     },
 
+    {
+        "name": "Gule Sider (Norwegian Yellow Pages)",
+        "url": "https://oppdater.gulesider.no/person",
+        "difficulty": "hard",
```
Comment (tupaschoal) [inline _data/sites.json:5229]:
```suggestion
        "difficulty": "impossible",
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**